Wikipedia/Code/WMFAppViewController.m

```diff
@@ -209,14 +221,72 @@ - (void)pauseApp {
     [self performHousekeepingIfNecessary];
 }
 
+-(void)handleSelectionOfShortcutItemIfNecessaryThen:(void (^)(BOOL shortcutWasHandled))completion{
+    UIApplicationShortcutItem *shortcutItemSelectedAtLaunch = [((AppDelegate*)[UIApplication sharedApplication].delegate) shortcutItemSelectedAtLaunch];
+    if (shortcutItemSelectedAtLaunch) {
+        
+        // Main UI may *not* have been loaded if the user...
```

Comment:
IMO, this means the shortcut items should be empty until the user goes through FTUE, obviating this logic. IOW, i don't think we want the user to circumvent FTUE via 3D touch shortcuts